ouroboros-network/src/Ouroboros/Network/PeerSelection/Governor/Monitor.hs

```diff
@@ -76,15 +87,26 @@ targetPeers PeerSelectionActions{readPeerSelectionTargets}
               targets,
               ledgerStateJudgement,
               bootstrapPeersFlag,
-              hasOnlyBootstrapPeers
-            } =
+              hasOnlyBootstrapPeers,
+              useGenesisFlag
+              } =
     Guarded Nothing $ do
-      targets' <- readPeerSelectionTargets
+      configurationTargets <- readPeerSelectionTargets
```

Comment:
@coot If the ```ConfigurationTargets``` are exported by the configuration module, then it is unlikely that it will be imported by upstream libraries as ```qualified PeerSelection``` and so what they'll get at best is Configuration.ConfigurationTargets and so that won't be an upgrade - unless you meant something else. I think it'll best if I export a newtype PeerSelectionConfigurationTargets from the configuration module, and it'll wrap a ConfigurationTargets type defined where PeerSelectionTargets are defined. Then internally in our library I can slog thru the code to change where PeerSelectionTargets are referred into ConfigurationTargets and define the use the function you propose to select the right targets based on the context. Is this about it?